generators/server/templates/_pom.xml

```diff
@@ -974,9 +977,6 @@
                     Enable the line below to have remote debugging of your application on port 5005
                     <jvmArguments>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
                     -->
-                    <arguments>
```

Comment:
This seems to be still required(atleast for prod profile). As there are weird issues without this during bootrun. Seems hikariCP is not loading prod profile when this is not set when using `-Pprod`. SInce having this doesnt cause any harm its better we leave it as it is and may be change it to `--spring.profiles.active=dev${profile.no-liquibase}${profile.no-swagger}`. same goes for prod profile and Gradle build.